packages/core/src/common/workspace-utils.ts

```diff
@@ -0,0 +1,37 @@
+/********************************************************************************
+  * Copyright (C) 2018 Ericsson and others.
+  *
+  * This program and the accompanying materials are made available under the
+  * terms of the Eclipse Public License v. 2.0 which is available at
+  * http://www.eclipse.org/legal/epl-2.0.
+  *
+  * This Source Code may also be made available under the following Secondary
+  * Licenses when the conditions for such availability set forth in the Eclipse
+  * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+  * with the GNU Classpath Exception which is available at
+  * https://www.gnu.org/software/classpath/license.html.
+  *
+  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+  ********************************************************************************/
+
+import { Path } from './path';
+
+export namespace WorkspaceUtils {
+
+    /**
+     * Returns shortened workspace path if applicable
+     * @param workspace
+     * @param home
+     */
+    export function getShortenedPath(workspace: string, home?: string): string {
```

Comment:
Rename the workspace variable to something else (like `path`), since it doesn't have to be a workspace path, it could the path of anything.

Reply:
I would suggest making the `home` parameter not `?`, since calling this function with `home == undefined` is pointless.  Though we still need to handle the case where the caller passes an empty string as the home.